tests/resume.test.ts

```diff
@@ -23,6 +23,35 @@ afterAll(async () => {
   await DatabaseConnection.close();
 });
 
+describe('resume fetching', () => {
+  test('only admin can fetch and only the visible resumes', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
+    const member = UserFactory.fake();
+    const resume = ResumeFactory.fake({ user: member, isResumeVisible: true });
+    const resumeInvisible = ResumeFactory.fake({ user: member, isResumeVisible: false });
+
+    await new PortalState()
+      .createUsers(admin, member)
+      .createResumes(member, resume, resumeInvisible)
+      .write();
+
+    const resumeController = ControllerFactory.resume(conn);
+
+    // throws permissions error for member
+    expect(resumeController.getAllVisibleResumes(member))
+      .rejects.toThrow(ForbiddenError);
+
+    // get response resumes
+    const expected = resume;
+    delete expected.user;
+    const response = await resumeController.getAllVisibleResumes(admin);
+    expect(response.error).toBeNull();
+    expect(response.resumes).toContainEqual(expected);
+    expect(response.resumes).toHaveLength(1); // only the visible resume is shown
+  });
```

Comment:
Instead of calling `delete` on objects to strip down the fields we don't want to test, let's instead make it clear the fields we do want to test. This follows the testing convention we have, and also doesn't add confusion on why we are calling `delete` on some seemingly arbitrary field. We can do something like:
```ts
const response = await resumeController.getAllVisibleResumes(admin);
expect(response).toStrictEqual({
  uuid: resume.uuid,
  user: member.uuid,
  isResumeVisible: resume.isResumeVisible
  ... // fill out for rest of fields
})
```